Makefile

```diff
@@ -106,9 +102,7 @@ clean-hub: ensure-operator-sdk
 	$(OPERATOR_SDK) cleanup cluster-manager --namespace open-cluster-management --timeout 10m
 
 cluster-ip:
-	cp $(KUBECONFIG) $(HUB_KUBECONFIG)
```

Comment:
I think we need to keep this otherwise we cannot run e2e automation in prow.

Reply:
e2e test is done.